### PR TITLE
cephfs-mirror: allow connecting to local cluster using mon address

### PR DESCRIPTION
**Pull request by vshankar**
Fixes: http://tracker.ceph.com/issues/50581
Signed-off-by: Venky Shankar <vshankar@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/tools/cephfs_mirror/Utils.cc [hunk 1]
```diff
@@ -40,12 +41,13 @@ int connect(std::string_view client_name, std::string_view cluster_name,
...
-  std::vector<const char*> args;
-  r = cct->_conf.parse_argv(args);
-  if (r < 0) {
-    derr << ": could not parse environment: " << cpp_strerror(r) << dendl;
-    cct->put();
-    return r;
+  if (!args.empty()) {
+    r = cct->_conf.parse_argv(args);
+    if (r < 0) {
+      derr << ": could not parse environment: " << cpp_strerror(r) << dendl;
```
Comment (leseb) [inline src/tools/cephfs_mirror/Utils.cc:47]:
```suggestion
      derr << ": could not parse command line args for " << cpp_strerror(r) << dendl;
```

**Review by leseb — COMMENTED** [review]
small clarification suggestion

File: src/tools/cephfs_mirror/Utils.cc (hunk 1 above)
Comment (vshankar) [reply src/tools/cephfs_mirror/Utils.cc:47]: ack

**Review by leseb — APPROVED** [review]

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
https://tracker.ceph.com/projects/cephfs/wiki/Master#2021-May-18

**Pull request merged**